diagnostics-app/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "diagnostics-app",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "~12.0.2",
+    "@angular/cdk": "^12.0.3",
+    "@angular/common": "~12.0.2",
+    "@angular/compiler": "~12.0.2",
+    "@angular/core": "~12.0.2",
+    "@angular/forms": "~12.0.2",
+    "@angular/material": "^12.0.3",
+    "@angular/platform-browser": "~12.0.2",
+    "@angular/platform-browser-dynamic": "~12.0.2",
+    "@angular/router": "~12.0.2",
+    "@angular/service-worker": "~12.0.2",
+    "rxjs": "~6.6.0",
+    "tslib": "^2.1.0",
+    "zone.js": "~0.11.4"
```

Comment:
This is used only for testing, should we put it under "dependencies"? Are there some sort of test dependencies?

Reply:
- `tslib` is a runtime library for TypeScript that contains all of the TypeScript helper functions.
- `zone.js` is essential to Angular for handling change detections. It allows Angular to know when the UI needs to be refreshed.

So both of them are core essentials of Angular!